web/cobrands/sass/_layout.scss

```diff
@@ -13,6 +13,11 @@ $header-top-border: $header-top-border-width solid $primary !default;
 
 $container-max-width: 60em !default;
 
+$primary_link_color: null !default;
+$primary_link_decoration: null !default;
```

Comment:
Do you need to define the two _decoration ones in here, they're not used

Reply:
Removed now! Thanks!